### DISCUSSION TITLE
Hey Can anyone tell me How to disable this 😑 its so annoying

### DISCUSSION THREAD

**Discussion by uuc110** [microsoft/vscode-discussions#299] [Extension Development QnA] [answered]
![image](https://user-images.githubusercontent.com/44457947/206843298-cf710b1e-cd2d-4885-8a64-d3f2f24bcb74.png)
I get this Ō when I press alt + shit + ctrl + o. And it's annoying me because I want to change my keybinding for enabling and disabling copilot but it's doing this with every keybinding and I want to disable this. symbol writing

**Comment by gjsjohnmurray** [comment, marked as answer]
See [#138339](https://github.com/microsoft/vscode/issues/138339)

I guess you are on Windows, and Windows is treating that keycombo as a way to enter an "international" character.
